components/eventing-controller/pkg/handlers/nats.go

```diff
@@ -122,7 +122,8 @@ func (n *Nats) SyncSubscription(sub *eventingv1alpha1.Subscription, cleaner even
 		}
 
 		for i := 0; i < subsConfig.MaxInFlightMessages; i++ {
-			natsSub, subscribeErr := n.connection.QueueSubscribe(subject, subject, callback)
+			queueGroupName := createKeyPrefix(sub) + string(types.Separator) + subject
```

Comment:
Could we add a comment why we introduced this prefix?

Reply:
Done